spec/robots/dor_repo/was_crawl_preassembly/build_was_crawl_druid_tree_spec.rb

```diff
@@ -22,7 +22,7 @@
     # let(:version_client) { instance_double(Dor::Services::Client::ObjectVersion, current: '1') }
 
     before do
-      # allow(WorkflowClientFactory).to receive(:build).and_return(workflow_client)
+      # allow(LyberCore::WorkflowClientFactory).to receive(:build).and_return(workflow_client)
```

Comment:
i know, global search and replace ... but ...

Reply:
De-crufted.